README.md

```diff
@@ -1,3 +1,127 @@
-# field_monitoring
+![](RackMultipart20220705-1-ruigrg_html_70e3b217e6f91606.png)
+
+# ATLAS Service Template Specification
+
+# Name: Crop Status Monitoring
+
+# Author(s): AgroApps (AGA)
+
+_The &quot;__Crop Status Monitoring&quot; ATLAS Service Template_
+
+This service will deliver a number of EO derived indicators for crop status monitoring consisting of:
+
+- Vegetation Indices
```

Comment:
Would it not be better to provide the list of possible indices as an Appendix (easily extendable) and concentrate more on generic wordings in the main body of the service template?